layouts/shortcodes/home-section-resources.html

```diff
@@ -0,0 +1,75 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Yi Liu <yi.liu@eclipse-foundation.org>
+
+  SPDX-License-Identifier: EPL-2.0
+-->
+
+<section class="container">
+  <h2>Resources</h2>
+  <div class="row">
+    <div class="col-md-6 col-sm-12">
+      <div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
+        <div class="flex-grow">
```

Comment:
Not sure we need to wrap this in an additional element, but it doesn't break anything. Just slightly messy.

Reply:
I think we need it. Since I wanna wrap the icons, titles and descriptions together to be in flex-grow div, so that the button will be squeezed to the bottom.

Reply:
Why not do this:
```
<div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
   <p><img class="img img-responsive margin-auto padding-20" style="height:150px;width:150px" src="images/resources/factsheet-icon.svg" alt="fact sheet icon"></p>
   <h3>Fact Sheet</h3>
   <p class="flex-grow">Get to know the Eclipse Foundation and learn about our strategic focus areas.</p>
   <p><a class="btn btn-secondary" href="https://www.eclipse.org/europe/eclipse-fact-sheet.pdf" aria-label="Download eclipse fact sheet pdf">Download</a></p>
</div>
```

It outputs the same look and feel, without extra wrapping (which is always good to avoid when you can). There is no semantic reason to say the container made up of those three elements should grow, when its just that the paragraph before the button should grow.

Reply:
This is a good idea! THX!